### PR TITLE
Earnings calculator responsive layout

### PR DESCRIPTION
**Pull request by dcgoodridge**
Responsive form
CSS improvements
Minor fixes

### CODE REVIEW

File: _sass/_calculator.scss [hunk 1]
```diff
@@ -5,49 +5,80 @@
...
 	color: red;
 }
-.editOption {
-    position: relative;
-    right: 47px;
-    bottom: 46px;
-    padding-left: 5px;
-    width: 95px;
+
+.calculator-otramoneda {
```
Comment (DannieBGoode) [inline _sass/_calculator.scss]: english

File: _pages/calculator.md [hunk 2]
```diff
@@ -9,67 +9,57 @@ sitemap: true
...
 				<option value="BTC"  min="2010-07-18">Bitcoin</option>
 				<option value="ETH"  min="2015-08-08">Ethereum</option>
 				<option value="LTC"  min="2013-09-15">Litecoin</option>
 				<option value="XMR"  min="2015-01-27">Monero</option>
 				<option value="DASH" min="2014-02-04">Dash</option>
 				<option value="XRP"  min="2015-01-30">Ripple</option>
-				<option class="editable">otro</option>
+				<option class="editable">Otra moneda...</option>
 			</select>
-			<input class="editOption" autofocus/>
```
Comment (DannieBGoode) [inline _pages/calculator.md]: why remove the autofocus?

File: _pages/calculator.md [hunk 3]
```diff
@@ -9,67 +9,57 @@ sitemap: true
...
-			<option value="XRP" min="2015-01-30">Ripple</option>
-		</datalist>
-	</span> -->
-	<p>
-		el día:
-		<input id="invest-date" type="date" value="2014-12-10" min="2010-07-18">
-
-		<div class="loader loader-calculator"></div>
-		<!-- <img src="../images/general/loading.gif"/> -->
-		<div id="calculator-results">
```
Comment (DannieBGoode) [inline _pages/calculator.md]: loader is needed

File: _pages/calculator.md (hunk 2 above)
Comment (dcgoodridge) [reply _pages/calculator.md]: Why do we have "autofocus" in a hidden form box?
I moved it to the first visible inpux box

File: _pages/calculator.md (hunk 3 above)
Comment (dcgoodridge) [reply _pages/calculator.md]: OK. Added again.

File: _sass/_calculator.scss (hunk 1 above)
Comment (dcgoodridge) [reply _sass/_calculator.scss]: Ok.

**Pull request merged**